scripts/issue/209/vote_to_drain_agent.py

```diff
@@ -0,0 +1,11 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    safe = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
```

Comment:
Not sure if missing something but the voting contract logic is hardcoded to require the vote() call from devMulti: https://etherscan.io/address/0xb7acd34643181c879437c2967538d5c0ea42b5d9#code#L686

Reply:
i think this is the relevant solidity function: https://etherscan.io/address/0xb935c3d80229d5d92f3761b17cd81dc2610e3a45#code#L1978

Reply:
That's true, I somehow ended in the TiemlockWithVoting contract. If interacting directly with the voting contract it should be fine.